tests/examples/pacer/nef/s3/cacb_3.json

```diff
@@ -4,28 +4,43 @@
   "court_id": "cacb",
   "dockets": [
     {
-      "case_name": "Maximo Arturo Arriola",
```

Comment:
Could we add this test example as a separate one to avoid overriding the original?